packages/fast-components-styles-msft/src/action-trigger/index.ts

```diff
@@ -0,0 +1,156 @@
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const contrastScale: number = designSystem.contrast;
+    const foregroundColor: string = designSystem.foregroundColor;
+    const backgroundColor: string = designSystem.backgroundColor;
+    const brandColor: string = designSystem.brandColor;
+    const direction: Direction = designSystem.direction;
+    const scaledNormalContrast: ContrastFunction = curry(normalContrast)(contrastScale);
+    const scaledEnsureNormalContrast: ContrastFunction = curry(ensureNormalContrast)(
+        contrastScale
+    );
+
+    // Define secondary button colors
+    const color: string = "white";
+    const secondaryBackgroundColor: string = scaledEnsureNormalContrast(
+        scaledNormalContrast(backgroundColor, foregroundColor),
+        color
+    );
+    const secondaryFocusBorderColor: string = scaledEnsureNormalContrast(
+        scaledEnsureNormalContrast(foregroundColor, backgroundColor),
+        secondaryBackgroundColor
+    );
+    const secondaryDisabledBackgroundColor: string = disabledContrast(
+        contrastScale,
+        secondaryBackgroundColor,
+        backgroundColor
+    );
+    const secondaryDisabledColor: string = disabledContrast(
+        contrastScale,
+        foregroundColor,
+        designSystem.brandColor
+    );
+    // Define primary button colors
+    const primaryRestBackgroundColor: string = scaledEnsureNormalContrast(
+        scaledEnsureNormalContrast(brandColor, backgroundColor),
+        color
+    );
+    const primaryDisabledBackground: string = disabledContrast(
+        contrastScale,
+        primaryRestBackgroundColor,
+        backgroundColor
+    );
+    const primaryDisabledColor: string = disabledContrast(
+        contrastScale,
+        color,
+        primaryDisabledBackground
+    );
+    const outlineColor: string = scaledEnsureNormalContrast(
+        foregroundColor,
+        backgroundColor
+    );
+    const outlineBorderColor: string = scaledNormalContrast(
+        foregroundColor,
+        backgroundColor
+    );
+    const outlineDisabledColor: string = disabledContrast(
+        designSystem.contrast,
+        outlineColor,
+        backgroundColor
+    );
+    const outlineDisabledBorderColor: string = outlineDisabledColor;
+    return {
+        actionTrigger: {
+            display: "inline-flex",
+            maxWidth: "100%",
+            minWidth: "14px",
+            border: "2px solid transparent",
+            lineHeight: "1",
+            textDecoration: "none",
+            whiteSpace: "nowrap",
+        },
+        actionTrigger_glyph: {
+            display: "inline-block",
+            position: "relative",
+            width: "8px",
+            [applyLocalizedProperty("marginRight", "marginLeft", direction)]: "6px",
+            marginTop: direction === Direction.ltr ? "4px" : "0",
+        },
+        actionTrigger__primary: {
+            "& $actionTrigger_glyph": {
+                fill: color,
```

Comment:
Question: What happens with these fill property values if the element is not an SVG?

Reply:
It's inert as far as I can tell, and elements that use "color" inherit the color property that button applies to it's content so seems to do the right thing.